### DISCUSSION TITLE
Resize font size

### DISCUSSION THREAD

**Discussion by leandro-benedet-garcia** [stoatchat/discussions#111] [Feature Suggestions]
It would be interesting to have a option to decrease the font size in the appearance menu.

I know creating custom css is a thing. But 90% of the users probably won't be programmers and even if they were, doesn't mean they know how to debug and edit css even if it's easy.

**Comment by necroTadpole** [comment]
Second this!! I hope this is something I can boost further because it would just be so convenient for people like me who are less than fully sighted :')
